packages/manager/tools/component-rollup-config/src/plugins/common/index.ts

```diff
@@ -0,0 +1,15 @@
+const translationNormalize = (text) => {
+  if (text) {
+    const filtered = text
+      .toString()
+      .replace(/&#13;\n/g, ' ') // carriage returns
```

Comment:
```suggestion
      .replace(/\r\n/g, ' ') // carriage returns
```

Reply:
same here the regex is about replacing the `&#13;` html code

Reply:
Why not replacing it why `\r` then ?